hudi-common/src/main/java/org/apache/hudi/common/table/log/block/HoodieHFileDataBlock.java

```diff
@@ -121,7 +122,7 @@ protected byte[] serializeRecords(List<HoodieRecord> records) throws IOException
     HFileContext context = new HFileContextBuilder()
         .withBlockSize(DEFAULT_BLOCK_SIZE)
         .withCompression(compressionAlgorithm.get())
-        .withCellComparator(new HoodieHBaseKVComparator())
+        .withCellComparator(ReflectionUtils.loadClass(KV_COMPARATOR_CLASS_NAME))
```

Comment:
We're going to get rid of this because this HFileContext is dependent on hadoop

Reply:
Yes.